server/src/main/java/org/elasticsearch/index/fielddata/plain/AtomicLongFieldData.java

```diff
@@ -56,7 +56,27 @@ public long ramBytesUsed() {
     public final ScriptDocValues<?> getLegacyFieldValues() {
```

Comment:
This function is unused since #30831. I'll open a separate pr to remove it.

Reply:
I left the code in here for now, but can also remove it, whatever you prefer

Reply:
I opened https://github.com/elastic/elasticsearch/pull/38087